db/migrations/04_add_entries_to_users_table.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO users(fullname,username,email,password)
```

Comment:
Was this part of the testing to check you could add entry to the table? Or to show the syntax behind adding it?